src/Components/basic/button/OptionButton.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import { theme } from '../../../styles/theme'
+
+interface OptionProps {
+  state: boolean
+  selected: boolean
+  onClick?: () => void
+  text: string
+}
+
+export const OptionBtn = (props: OptionProps) => {
+  return (
+    <OptionBtnComponent onClick={props.onClick} state={props.state} selected={props.selected}>
+      {props.text}
+    </OptionBtnComponent>
+  )
+}
+
+const OptionBtnComponent = styled.div<{ state: boolean; selected: boolean }>`
+  ${({ state, selected }) => css`
+    ${theme.typography.b02};
+    background-color: ${theme.palette.background.light};
+    padding: 16px;
+    margin-bottom: 8px;
+    border-radius: 6px;
+    border: 1.5px solid rgba(0, 0, 0, 0);
```

Comment:
you can also define border color in your design system
```suggestion
    border: 1.5px solid ${theme.palette.border.black};
```

Reply:
Actually, it is for transparent color not to change the size of components when changing the state(before & after clicking). THen, I need to add the color.